src/main/java/org/eclipse/basyx/submodel/restapi/SubmodelElementProvider.java

```diff
@@ -117,7 +118,8 @@ public void setValue(String path, Object newValue) throws ProviderException {
 			throw new MalformedRequestException("The given path '" + path + "' does not end in /value.");
 		}
 
-		if (!specializedProvider && path.equals(MultiSubmodelElementProvider.VALUE)) {
+		if (!specializedProvider && path.equals(MultiSubmodelElementProvider.VALUE)
+				&& !isFileInputStream(newValue)) {
```

Comment:
Why is this check for setValue necessary? We're using createValue for the upload